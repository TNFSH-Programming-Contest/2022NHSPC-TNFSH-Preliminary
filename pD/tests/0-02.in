4 5
1 1 0 2
2 0 0 2 0
.....
T.T..
T..T.
.....
