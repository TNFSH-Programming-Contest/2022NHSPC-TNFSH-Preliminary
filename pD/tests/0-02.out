A....
T.TA.
T..T.
A..A.
